client/marshaller/kryo-marshaller/src/main/java/org/infinispan/marshaller/kryo/KryoTranscoder.java

```diff
@@ -0,0 +1,55 @@
+package org.infinispan.marshaller.kryo;
+
+import net.spy.memcached.CachedData;
+import net.spy.memcached.transcoders.Transcoder;
+
+/**
+ * @author Ryan Emerson
+ * @since 9.0
+ */
+public class KryoTranscoder implements Transcoder<Object> {
```

Comment:
If you want to test it that's fine but we should not expose this transcoder stuff to the user.